draft-ietf-webpush-protocol.xml

```diff
@@ -1180,7 +1180,10 @@ HEADERS      [stream 82] +END_STREAM
         <t>
           However, reliability might not be necessary for messages that are
           transient (e.g. an incoming call) or messages that are quickly
-          superceded (e.g. the current number of unread emails).
+          superceded (e.g. the current number of unread emails).  Though a push
+          service is encouraged to provide negative acknowledgments (see <xref
+          target="receive_receipts"/>), push messages can be lost if the push

```

Comment:
This broke the build. Need to change to _target="receive_receipt"_

"encouraged" is a bit weak, since the actual language is a MUST when delivery receipts are requested.